src/week1/day0/css/style.css

```diff
@@ -0,0 +1,72 @@
+body{
+   font-size: 14px;
+   font-family: serif;
+}
+
+.container1{
+   display: flex;
+   flex-flow: row;
+}
+
+header{
+   background: black;
+   color: white;
+   text-align: center;
+}
+
+nav{
+   width: 25%;
+   background: orange;
+   color: white;
+}
+
+section{
```

Comment:
always prefer class instead of element selector, or at least (for compound selectors) don't start with an element, always with a class. E.g: .some-class ele { }